usaspending_api/database_scripts/matview_generator/mv_contract_award_search.json

```diff
@@ -50,27 +69,57 @@
     "  TFA.toptier_code AS funding_toptier_agency_code,",
     "  SAA.subtier_code AS awarding_subtier_agency_code,",
     "  SFA.subtier_code AS funding_subtier_agency_code,",
+    "  CASE",
+    "    WHEN TFA.name IS NOT NULL",
+    "      THEN CONCAT(",
+    "        '{\"name\":\"', TFA.name,",
+    "        '\",\"code\":\"', TFA.toptier_code,",
+    "        '\",\"id\":\"', (SELECT a1.id FROM agency a1 WHERE a1.toptier_agency_id = (SELECT a2.toptier_agency_id FROM agency a2 WHERE a2.id = latest_transaction.funding_agency_id) ORDER BY a1.toptier_flag DESC, a1.id LIMIT 1), '\"}'",
+    "      )",
+    "    ELSE NULL",
+    "  END AS funding_toptier_agency_agg_key,",
+    "  CASE",
+    "    WHEN SFA.name IS NOT NULL",
+    "      THEN CONCAT(",
+    "        '{\"name\":\"', SFA.name,",
+    "        '\",\"code\":\"', SFA.subtier_code,",
+    "        '\",\"id\":\"', (SELECT a1.id FROM agency a1 WHERE a1.toptier_agency_id = (SELECT a2.toptier_agency_id FROM agency a2 WHERE a2.id = latest_transaction.funding_agency_id) ORDER BY a1.toptier_flag DESC, a1.id LIMIT 1), '\"}'",
+    "      )",
+    "    ELSE NULL",
+    "  END AS funding_subtier_agency_agg_key,",
     "",
     "  rl_country_lookup.country_code AS recipient_location_country_code,",
     "  rl_country_lookup.country_name AS recipient_location_country_name,",
     "  transaction_fpds.legal_entity_state_code AS recipient_location_state_code,",
+    "  RL_STATE_LOOKUP.name AS recipient_location_state_name,",
+    "  RL_STATE_LOOKUP.fips AS recipient_location_state_fips,",
+    "  RL_STATE_POPULATION.latest_population AS recipient_location_state_population,",
```

Comment:
I don't see these fields in the original view. What will they help?

Reply:
Removed `recipient_location_state_name`, `recipient_location_state_fips`, `recipient_location_state_population`, `recipient_location_county_population`, `recipient_location_congressional_population`, `pop_state_name`, `pop_state_fips`, `pop_state_population`, `pop_county_population`, and `pop_congressional_population`. (Same as above in the delta view)